dags/common/immersion_facilitee/helpers.py

```diff
@@ -61,6 +61,14 @@ def get_all_items(path: str) -> list[dict]:
 
         data_partial = response.raise_for_status().json()
 
+        if len(data_partial) > 99:
+            logger.warning(
+                "More than 99 items found for date %s: %d items. "
+                "This may indicate that we do not get all data because of the lack pagination.",
+                str_date_g,
+                len(data_partial),
+            )
```

Comment:
```suggestion
        if len(data_partial) >= MAX_API_ITEMS_RETURNED:
            logger.warning(
                "More than %s items found for date %s: %d items. "
                "This may indicate that we do not get all data because of the lack of pagination.",
                MAX_API_ITEMS
                str_date_g,
                len(data_partial),
            )
```
Utilisation d'une constante pour éviter un _magic number_ et ainsi documenter plus précisément d'où vient cette limitation.

Et correction d'une typo dans le log.